packages/holocron-module-register-webpack-plugin/__tests__/HolocronModuleRegisterPlugin.spec.js

```diff
@@ -98,11 +98,37 @@ describe('HolocronModuleRegisterPlugin', () => {
       const fileContents = fs.readFileSync(path.join(buildPath, outputFileName)).toString();
       expect(fileContents.startsWith('(function() {')).toBe(true);
       expect(fileContents).toContain('const ModuleWithAsyncImport = () =>');
-      expect(fileContents.endsWith('Holocron.registerModule("some-module", holocronModule);})();')).toBe(true);
+      expect(fileContents.endsWith('Holocron.registerModule("some-module", holocronModuleName);})();')).toBe(true);
       const asyncChunkContents = fs.readFileSync(path.join(buildPath, `async-import.${outputFileName}`)).toString();
       expect(asyncChunkContents).toContain('() => \'Hello, world\'');
       expect(asyncChunkContents).not.toContain('Holocron.registerModule("some-module"');
       done();
     });
   });
+
+  it('should registerModule with provided `moduleName` and `holocronModuleName`', (done) => {
+    expect.assertions(1);
+    const outputFileName = 'webpack-test-output-async.js';
+    const moduleName = 'some-module';
+    const holocronModuleName = `holocronModule-${moduleName}`;
+    webpackOptions = {
```

Comment:
we should avoid mutation, instead, you can keep `webpackOptions` as a constant and make a new var called `webpackOptionsWithPlugins`

Reply:
Agreed!